docs/UserGuide.md

```diff
@@ -3,14 +3,47 @@ layout: page
 title: User Guide
 ---
 
-Fine\$\$e is a **desktop finance tracker optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, Fine\$\$e can help you track your transactions (expenses/incomes) and cultivate good financial habits faster than traditional GUI apps.
-
 * Table of Contents
 {:toc}
 
 --------------------------------------------------------------------------------------------------------------------
+## 1. Introduction
+
+### 1.1 About Fine$$e
+Welcome to Fine$$e!
+
+Looking for an all-in-one solution to help you develop good financial habits? Look no further!
+
+Fine$$e is an integrated platform fully customized for tertiary and university students with the aim of helping you track your finances effectively. Fine$$e allows you to keep track of your incomes, expenses and savings with a few simple commands. Furthermore, to help you cultivate good financial habits Fine$$e allows you to set expense limits and savings goals.
```

Comment:
Add line breaks after every full stop. This will help us version control the document better. Same for the rest of the document.